examples/generate_computers_excel_report.ps1

```diff
@@ -0,0 +1,68 @@
+<#
+.SYNOPSIS
+        Cyberwatch API script to generate an Excel report of Computers' information monitored by Cyberwatch.
+.DESCRIPTION
+        This script serves as an example, it can and should be adapted to your needs.
+        This example:
+            - defines an filter based on environments;

```

Comment:
based on the assets criticalities (environment is the backend name, but should be translated to criticality each time we use it in a sentence)